frontend/default/routes.py

```diff
@@ -11,9 +14,29 @@ def index():
 
 @default_bp.errorhandler(404)
 def not_found(error):
-    return render_template("404.html"), 404
+    current_app.logger.error(f"Encountered 404: {error}")
+    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
+    return render_template(
+        "404.html",
+        contact_us_phone=round_data.contact_details["phone"],
+        contact_us_email_address=round_data.contact_details["email_address"],
+        contact_us_text_phone=round_data.contact_details["text_phone"],
+        opening_time=round_data.support_availability["time"],
+        opening_days=round_data.support_availability["days"],
+        closed=round_data.support_availability["closed"]
+    ), 404
 
 
 @default_bp.errorhandler(500)
 def internal_server_error(error):
-    return render_template("500.html"), 500
+    current_app.logger.error(f"Encountered 500: {error}")
+    round_data = get_round_data(Config.DEFAULT_FUND_ID, Config.DEFAULT_ROUND_ID, as_dict=True)
```

Comment:
Need to handle error in/from `get_round_data` and remove network call in 500 page